code/game/objects/items/RCD.dm

```diff
@@ -1109,13 +1107,46 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	///Current selected color, for ducts
 	var/current_color = "omni"
 
+
 /obj/item/construction/plumbing/Initialize(mapload)
 	. = ..()
+
+	//design types supported for this plumbing rcd
 	set_plumbing_designs()
 
-/obj/item/construction/plumbing/examine(mob/user)
-	. = ..()
-	. += span_notice("Alt-Click to change layer and duct color.")
+	//set cost of each machine & initial blueprint
+	for(var/obj/machinery/plumbing/plumbing_type as anything in plumbing_design_types)
+		machinery_data["cost"][plumbing_type] = plumbing_design_types[plumbing_type]
+	blueprint =  plumbing_design_types[1]
+
+/obj/item/construction/plumbing/proc/set_plumbing_designs()
+	plumbing_design_types = list(
+		//category 1 Synthesizers i.e devices which creates , reacts & destroys chemicals
+		/obj/machinery/plumbing/synthesizer = 15,
+		/obj/machinery/plumbing/reaction_chamber/chem = 15,
+		/obj/machinery/plumbing/grinder_chemical = 30,
+		/obj/machinery/plumbing/growing_vat = 20,
+		/obj/machinery/plumbing/fermenter = 30,
+		/obj/machinery/plumbing/liquid_pump = 35,   //extracting chemicals from ground is one way of creation
```

Comment:
Do not align comments. Only use one space, no tabs

Reply:
Got it